bedrock/firefox/views.py

```diff
@@ -657,7 +656,7 @@ class NewView(VariationMixin, L10nTemplateView):
     ]
 
     # place expected ?v= values in this list
-    variations = ["a", "b"]
+    variations = ["a", "b", "fx94"]
```

Comment:
Curious: what's the significance of removing the `template_context_variations` in `urls.py` and adding the variation instead in the L10nTemplateView class in `views.py`?

Reply:
Seeing as this is already an established view, complete with variation logic, adding in `L10nTemplateView` as a mixin seemed like it was over complicating things (variation context variables were being handle in two separate ways)